boot/seal.go

```diff
@@ -153,8 +158,12 @@ func resealKeyToModeenv(rootdir string, model *asserts.Model, modeenv *Modeenv)
 	if err != nil {
 		return fmt.Errorf("cannot find the recovery bootloader: %v", err)
 	}
-
-	recoveryBootChains, err := recoveryBootChainsForSystems(modeenv.CurrentRecoverySystems, rbl, model, modeenv)
+	tbl, ok := rbl.(bootloader.TrustedAssetsBootloader)
+	if !ok {
+		// nothing to do
```

Comment:
I made this an internal error now that we know whether we have sealed keys at all or not